src/utils/openvpn.ts

```diff
@@ -398,10 +400,6 @@ export class VpnManager extends EventEmitter implements VpnManagerEvents {
 		await this.exec('hold release');
 	}
 
-	public async getStatus() {
-		await this.exec('status');
-	}
```

Comment:
Why are we dropping this?

Reply:
Nevermind, I see that it returns `void`. Since its meant to get and return a status (like the call in `vpn-worker.ts` indicates), but is not and has not, then its not useful to keep around.